GAME/NEWINV2.C

```diff
@@ -10,6 +10,8 @@
 #include "CD.H"
 #include "LARA2GUN.H"
 #include "LARA1GUN.H"
+#include "../SPEC_PSX/PSXINPUT.H"
```

Comment:
Should be wrapped in #if PSX_VERSION spec_psx path not required since it's automatically an include directory so #include "PSXINPUT.H" should do the trick.